package/src/components/Message/MessageSimple/MessageReplies.tsx

```diff
@@ -46,16 +40,11 @@ const styles = StyleSheet.create({
     fontSize: 12,
     fontWeight: '700',
     paddingBottom: 5,
-    paddingLeft: 8,
+    paddingHorizontal: 8,
   },
   rightMessageRepliesCurve: {
     borderBottomRightRadius: 16,
-    borderLeftColor: 'transparent',
-    ...Platform.select({
-      android: {
-        borderLeftWidth: 0,
-      },
-    }),
+    borderLeftWidth: 0,
```

Comment:
If its 0, if there a need to have this property in the styles anymore?

Reply:
Yes, the current structure first passes `styles.messageRepliesCurve` with a `borderWidth: 1`. Then, we remove the opposite side border within each `rightMessageRepliesCurve` or `leftMessageRepliesCurve`. I've attached an image if we just remove that line. 

<img width="239" alt="Screenshot 2023-09-03 at 14 38 34" src="https://github.com/GetStream/stream-chat-react-native/assets/45667737/8b59c17b-2bbd-40e4-ab9e-7d2c903f7415">

However, if this is not ideal, we could remove the `borderWidth: 1` from the `styles.messageRepliesCurve` and replace it with `borderWithBottom: 1` and then have a left or right borderWidth with each replies curve. This should give the same impact.